build.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# prevent sourcing of this script, only allow execution
+$(return >/dev/null 2>&1)
+test "$?" -eq "0" && { echo "This script should only be executed." >&2; exit 1; }
+
+# exit on errors, undefined variables, ensure errors in pipes are not hidden
+set -Eeuo pipefail
+
+declare mydir
+mydir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
+
+build_ts_utils() {
+    tsc -p ${mydir}/packages/utils/tsconfig.json
+}
+
+build_ts(){
+    yarn workspace @hoprnet/hopr-ethereum run build:sol:types
+    tsc --build tsconfig.build.json
+    yarn workspace @hoprnet/hoprd run buildAdmin
+}
+
+build_rs() {
+    rustup target add wasm32-unknown-unknown
+    cargo build --release --target wasm32-unknown-unknown
```

Comment:
Worth explaining here why the Rust code is compiled once using `cargo build` and for the second time as a part of the `build:wasm` action. The devs are not supposed to be looking this up in the history of the PR descriptions.

Reply:
Done by https://github.com/hoprnet/hoprnet/pull/3853